template/docroot/sites/default/settings.php

```diff
@@ -733,10 +733,20 @@
  * Keep this code block at the end of this file to take full effect.
  */
 if ($is_local_env) {
+  // Load Dev Desktop settings.
   if (isset($_SERVER['DEVDESKTOP_DRUPAL_SETTINGS_DIR']) && file_exists($_SERVER['DEVDESKTOP_DRUPAL_SETTINGS_DIR'] . '/loc_${project.machine_name}_dd.inc')) {
     require $_SERVER['DEVDESKTOP_DRUPAL_SETTINGS_DIR'] . '/loc_${project.machine_name}_dd.inc';
   }
+  // Load local machine settings.
   elseif (file_exists(DRUPAL_ROOT . '/sites/default/settings/local.settings.php')) {
     require DRUPAL_ROOT . '/sites/default/settings/local.settings.php';
   }
+
+  // Load Travis CI settings.
+  if (getenv('TRAVIS') && file_exists(DRUPAL_ROOT . '/sites/default/settings/travis.settings.php')) {
+    require DRUPAL_ROOT . '/sites/default/settings/travis.settings.php';
+  }
+  elseif (getenv('TUGBOAT_URL') && file_exists(DRUPAL_ROOT . '/sites/default/settings/tugboat.settings.php')) {
```

Comment:
This is not Travis, correct? We should have a comment above this line ('Load Tugboat settings') if we're commenting 'Load Travis CI settings' at the start of this conditional.